src/Connection/Internal/Http2ConnectionProcessor.php

```diff
@@ -732,7 +717,7 @@ public function handleStreamException(Http2StreamException $exception): void
         $exception = new SocketException($exception->getMessage(), $code, $exception);
 
         if ($code === Http2Parser::REFUSED_STREAM) {
-            $exception = new UnprocessedRequestException($exception);
+            $exception = new UnprocessedRequestException($exception); // TODO
```

Comment:
There's still a TODO here, something missing yet?

Reply:
The branch gets rid of `UnprocessedRequestException` mostly, but not entirely, yet. Not sure why I mixed those changes into one branch.